mlrun/api/api/endpoints/model_endpoints.py

```diff
@@ -363,9 +393,31 @@ def _verify_endpoint(project, endpoint_id):
         )
 
 
-def get_endpoint_id(endpoint: ModelEndpoint) -> str:
-    endpoint_unique_string = (
-        f"{endpoint.spec.function}_{endpoint.spec.model}_{endpoint.metadata.tag}"
-    )
-    md5 = hashlib.md5(endpoint_unique_string.encode("utf-8")).hexdigest()
-    return f"{endpoint.metadata.project}.{md5}"
+def get_model_endpoint_secrets(_request: Request):
+    access_key = _request.headers.get("X-V3io-Session-Key")
+    if not access_key:
+        raise MLRunBadRequestError(
+            "Request header missing 'X-V3io-Session-Key' parameter."
+        )
+
+    v3io_api = environ.get("V3IO_WEBAPI_PORT_8081_TCP")
+    if not v3io_api:
+        raise MLRunPreconditionFailedError(
+            "Environment missing 'V3IO_WEBAPI_PORT_8081_TCP' parameter."
+        )
+    else:
+        v3io_api = v3io_api.replace("tcp", "http")
+
+    v3io_framesd = environ.get("FRAMESD_PORT_8081_TCP")
+    if not v3io_framesd:
+        raise MLRunPreconditionFailedError(
+            "Environment missing 'V3IO_WEBAPI_PORT_8081_TCP' parameter."
+        )
+    else:
+        v3io_framesd = v3io_framesd.replace("tcp", "http")
```

Comment:
This counts on the fact that there is a svc (k8s service) named `v3io-webapi` - the svc name is originated from the app service name, we don't want to this coupling. (same for framesd)
Please simply... started describing what I want but understood it's a bit complicated - done that in https://github.com/mlrun/mlrun/pull/691 - so here you'll simply do `config.httpdb.v3io_api/v3io_framesd`